tracer/src/Datadog.Trace.Coverage.collector/FiltersHelper.cs

```diff
@@ -87,15 +87,15 @@ public static bool FilteredBySourceFile(string sourcePath, IReadOnlyList<string>
                             }
                         }
 
-                        instance.AddInclude(Path.IsPathRooted(filter) ? filter.Substring(Path.GetPathRoot(filter).Length) : filter);
+                        instance.AddInclude(Path.IsPathRooted(filter) ? filter.Substring(Path.GetPathRoot(filter)?.Length ?? 0) : filter);
```

Comment:
I think _technically_ these aren't necessary because `Path.GetPathRoot` will be non null if `Path.IsPathRooted` returns true.

In fact, `Path.GetPathRoot()` internally calls `Path.IsPathRooted` so it's probably better to rewrite these as:

```suggestion
                        instance.AddInclude(Path.GetPathRoot(filter) is { } rooted ? filter.Substring(rooted.Length) : filter);
```